js/ui/collection/ui.collection_widget.edit.js

```diff
@@ -50,12 +50,12 @@ const CollectionWidget = BaseCollectionWidget.inherit({
             selectionRequired: false,
 
             /**
-            * @name CollectionWidgetOptions.selectionByClick
+            * @name CollectionWidgetOptions.selectByClick
```

Comment:
Looks like this doc comment doesn't use anymore and we can remove it. It option is not public and we declare others doccomments for 'selectByClick' option on widgets' level.

Reply:
https://github.com/DevExpress/DevExtreme/pull/23941/commits/0577b0919d700606c37e21a37c9d0f69db7c5a17